### ISSUE TITLE
Example usage need to be updated

### ISSUE THREAD

**Issue by ppmim** [jmccormac01/Donuts#42]
> > > for image in science_image_names:
> > > ...     x, y = d.measure_shift(checkimage=image)
> > > ...     print(x, y)

to something as:

> > > for image in science_image_names:
> > > ...     res = d.measure_shift(checkimage_filename=image)
> > > ...     print(res.x, res.y)

**Comment by mindriot101** [comment]
Hi thanks for pointing this out to us. This has actually been fixed in the [development version docs](http://donuts.readthedocs.io/en/devel/donuts/user_guide.html) but we didn't bother to bump the version for this. Sorry for the confusion! We don't want to confuse users, so we'll push a new version soon.

**Comment by ppmim** [comment]
Great! but don't forget to update 'checkimage_filename=image'.

**Comment by jmccormac01** [comment]
Thanks for the input! Ok, I think I've got to the bottom of the RTD error. The definitions for `latest`, `stable`, and `default` were a bit of a mess. I've set `default` to follow `master` and `latest` to follow `devel`. `stable` gets picked up automatically if there is a stable tag. We should tag `0.2.1` as stable after the release for completeness.

**Comment by ppmim** [comment]
I'm developing a integrated prototype to test donuts in our telescope at [OSN](http://www.osn.iaa.es). I'll let you know about our results.

**Comment by jmccormac01** [comment]
Great! I hope it works well for you. At NGTS and NITES we implemented two additional methods at the backend of DONUTS. We will proobably describe these in a follow-up paper soon.  Briefly, they are:

1) A PID control loop to cope with mechanical in accuracies
2) Measurement buffer to filter out spurious measurements (satellite streaks etc). 

A simple PID loop in Python can be found [here](http://code.activestate.com/recipes/577231-discrete-pid-controller/), I implemented PIDx and PIDy to treat the axes separately. 

The measurement buffer is a numpy array (one for x and one for y) which contains the last 30 shifts. If the newest shift in x or y is greater than 5sigma of the measurement buffer ignore that shift, but put the measurement in the buffer (so real large shifts get accounted for after a few images, once they affect the sigma in the buffer).

I hope this helps. We've kept this new stuff out of DONUTS for now as it is quite hardware specific. Once we write it up properly we can add it in a generalise it.
